### DISCUSSION TITLE
A few CI jobs have been stuck for 4 months

### DISCUSSION THREAD

**Discussion by nschonni** [mastodon/mastodon#22332] [General]
Since a bunch of PRs got merged today, there is a little bit of a build queue. When looking at the running jobs https://github.com/mastodon/mastodon/actions?query=is%3Ain_progress i noticed these 2 that have been running for months now:
1. https://github.com/mastodon/mastodon/actions/runs/2929851771
2. https://github.com/mastodon/mastodon/actions/runs/2929850783

Likely causes by an issue on the GitHub end, as jobs should timeout what ahead of that. It might be possible to manually cancel them, or escalate to GitHub if not.
I think they may be taking up some of your free build queue slots for now

**Comment by nightpool** [comment]
Unfortunately I get this error when trying to cancel the workflow: 
<img width="1633" alt="image" src="https://user-images.githubusercontent.com/233815/215574740-5ee123f6-4f4f-46bc-9e42-6d4fdbbcb2ac.png">

**Reply by nightpool** [reply]
Issue escalated to Github support: https://support.github.com/ticket/personal/0/1990296

**Reply by nschonni** [reply]
Awesome!

**Reply by nschonni** [reply]
Looks like they got cleaned up!

**Comment by nschonni** [comment]
Looks like there are 2 more that got stuck queueing https://github.com/mastodon/mastodon/actions?query=is%3Aqueued
Probably GitHub hiccuped and they got stuck because the PR landed
